python/tskit/_version.py

```diff
@@ -1,4 +1,4 @@
 # Definitive location for the version number.
 # During development, should be x.y.z.devN
 # For beta should be x.y.zbN
-tskit_version = "0.3.0.b1"
+tskit_version = "0.3.0.b2"
```

Comment:
Nope, ``0.3.0b2``!

Reply:
Wow, my brain has packed up and migrated north due to the heat. Fixed, hopefully!